.github/workflows/docker.yml

```diff
@@ -21,8 +21,18 @@ jobs:
         image:
           - name: mariadb
             context: ./containers/mariadb
-            target: ''
-            version: ''
+            target: 'mariadb'
+            version: '10.10'
+            build-args: ''
+          - name: mariadb
+            context: ./containers/mariadb
+            target: 'mariadb_106'
```

Comment:
For my understanding, why are we using different targets here, not one target with different versions like we did with the PHP containers?